test/testsuite/linalg.jl

```diff
@@ -116,7 +104,7 @@
             n = 128
             d = AT(rand(Float32, n))
             D = Diagonal(d)
-            F = AT(collect(D))
+            F = collect(D)
```

Comment:
Prevent from calling dense `cholesky` method, which is not available for `oneArrays`